### DISCUSSION TITLE
Question about Mailpit

### DISCUSSION THREAD

**Discussion by essovius** [coollabsio/coolify#4510] [Q&A]
![Screenshot_20](https://github.com/user-attachments/assets/09ce04b3-b52a-4a4c-996d-3a185bad4b23)

I just setup mailpit, How can I connect to the bottom link that ends with `1025` I allowed the port in my firewall settings.

**Comment by LucaNerlich** [comment]
Hi, the first link is the mailpit dashboard.
The second is your coolify server and the smtp port.

You will need to configure your email setup to send to this domain (yourdomain.com:1025). Depending on your setup, do not add "https://" do the mail server host config.


e.g im working with AEM and this is my dev setup

```json
{
    "smtp.host": "coolify.mydomain.com",
    "smtp.port": "1025",
    "smtp.user": "",
    "smtp.password": "",
    "from.address": "noreply@test.com",
    "smtp.ssl": false,
    "smtp.starttls": false,
    "smtp.requiretls": false,
    "debug.email": false,
    "oauth.flow": false
}
```

works great:
![image](https://github.com/user-attachments/assets/2306061c-ded4-458d-8968-7c9a587e87ed)
